api/v1beta1/common_types.go

```diff
@@ -43,6 +43,10 @@ const (
 	DBPurgeDefaultAge = 30
 	//DBPurgeDefaultSchedule is in crontab format, and the default runs the job once every day
 	DBPurgeDefaultSchedule = "1 0 * * *"
+	//CleanerDefaultSchedule is in crontab format, and the default runs the job once every day
```

Comment:
I think it should be runs after every 30 minutes?

Reply:
ack thanks, a copy/paste leftover, I'll fix it in my next PS

Reply:
Done